master/internal/api_runs.go

```diff
@@ -140,6 +140,7 @@ func getRunsColumns(q *bun.SelectQuery) *bun.SelectQuery {
 		Column("r.external_run_id").
 		Column("r.project_id").
 		Column("r.searcher_metric_value").
+		ColumnExpr("CONCAT(p.key, '-' ,CAST(r.local_id as text)) as local_id").
```

Comment:
question (non-blocking): do we prefer CAST(...) or `r.local_id::text` from a style perspective?

Reply:
Changed to `::text` as I believe this is the style used elsewhere.

Reply:
I don't think there's a hard preference. Do what you like!